doc/source/deployment/setup-deployer.rst

```diff
@@ -63,17 +63,27 @@ We currently only allow to deploy master branch from sources.
 
 1. (For developers only) Clone the repository.
 
-   The following software must be manually installed on your `Deployer` using zypper or pip install:
+   The following software must be manually installed on your `Deployer` using zypper or pip install.
+   Enable following repos to install packages:
 
      * ansible >= 2.8.3
      * git-core
      * jq
+     * python
      * python3-virtualenv
-     * python3-jmespath
-     * python3-netaddr
-     * python3-openstacksdk
+       SUSEConnect --product sle-module-desktop-applications/15.1/x86_64
```

Comment:
So this is mixing in commands with a list of required software/packages. Instead should we move this to a section above (or even below) that gives a few preparation steps followed by a zypper one-liner to install all of these?

Reply:
I think this is more clear information of why we need to enable those repos.  Like  which package belong to which repo.

Reply:
This section is simply listing the requirements, not how to install them. This is because in some cases the requirements may be installed with pip or from other sources/repositories.

So I think if we have a second section that shows how to install them with the actual zypper commands and SUSEConnect commands (maybe with a common on why SUSEConnect is needed) that would be helpful, but we shouldn't mix the two.